app/models/conference.rb

```diff
@@ -4,7 +4,7 @@ class Conference < ActiveRecord::Base
   require 'uri'
   serialize :events_per_week, Hash
   # Needed to call 'Conference.with_role' in /models/ability.rb
```

Comment:
Add another line of comment briefly explaining why we need the options to resourcify